specification/trace/semantic_conventions/cloudevents.md

```diff
@@ -0,0 +1,51 @@
+# CloudEvents
+
+This document defines how to apply semantic conventions when instrumenting code that uses [CloudEvents](https://cloudevents.io/).
+These semantic conventions are general and not tied to any protocol used to send/receive the events.
```

Comment:
"General" conventions are conventions that should be used with another "main" type of span and thus usually do not prescribe a name or kind. I don't think this is what you meant.
I would leave out this sentence entirely, as freestanding semantic conventions are more or less the default anyway.

Reply:
This could be reworded to explicitly point out this is only about the event (payload) and that transport would be represented by a separate span depending on the underlying protocol used.

Reply:
Good point, I also don't like the way it is. Will try to come up with something close to what @arminru  mentioned. I think it's valid to emphasize that this is not related to the transport.

Reply:
Reworked on this part, hopefully now it's clear that it's only talking about sending/receiving the events.